app/views/layouts/application.html.haml

```diff
@@ -14,7 +14,7 @@
     %meta{:name => "author", :content => "Diaspora, Inc."}/
 
     %link{:rel => 'shortcut icon', :href => "#{image_path('favicon.png')}" }
-    %link{:rel => 'apple-touch-icon', :href => "#{image_path('apple-touch-icon.png')}"}
+    %link{:rel => 'apple-touch-icon', :href => "#{image_path('favicon.png')}"}
```

Comment:
> If someone has a better solution

A favicon should be... a favicon. apple-touch-icon is not a favicon, it is the icon used when the user pins the website as a shortcut on his mobile. So we hould have different property here.

```
<link rel="icon" type="image/x-icon" href="(...)favicon.ico"> <!-- The favicon -->
<link rel="apple-touch-icon" href="(...)apple-touch-icon.png"> <!-- The icon used on iphone, also used by Android -->
<meta name="msapplication-TileImage" content="(...).png" /> <!-- the one used by windows 8 --->
```

In reality, there is a lot more properties that just those. But if the apple-touch-icon is used as the favicon, it's definitely a bug. But maybe you don't have to deal with that in this PR ;)